app/config.go

```diff
@@ -54,6 +54,7 @@ func (a *App) LoadConfig(configFile string) *model.AppError {
 	a.configFile = configPath
 
 	utils.ConfigureLog(&cfg.LogSettings)
+	l4g.Info("Using config file at %s", configPath)
```

Comment:
This, just to prove to myself that the platform binary wasn't loading something else that explained the odd behaviour.